### ISSUE TITLE
Accessibility issues in HighCharts for text and images of text not providing sufficient contrast

### ISSUE THREAD

**Issue by pdaryamane** [highcharts/highcharts#21867]
#### Expected behaviour
Elements created inside Highcharts should pass accessibility standards

#### Actual behaviour
 Accessibility issue highlighted by Access Assistant (Google Chrome extension) that we need to ensure text and images of text provide sufficient contrast. 
The areas pointed out: 

- desc tag which highcharts adds <desc aria-hidden="true">Created with Highcharts 11.4.7</desc>.
**Error:** "This desc contains text with a background color of #2b2c2e rgb(43, 44, 46) and foreground color of #212529 rgb(33, 37, 41) that is less than 18 point in size; or bold text less than 14 point in size that has a luminosity contrast ratio of 1.104, which is below 4.5:1". 

Other elements where the issue is observed : text, tittle, tspan in Highchart World Map for country name, zoom in, zoom out buttons. 

#### Live demo with steps to reproduce
A highcharts map can be used to check the issue. Attaching a snapshot of the same map and Access Assistant results :
![image](https://github.com/user-attachments/assets/d65bf2cf-906c-4acc-83c3-3c79b67e3ea2)

#### Product version

highcharts: 11.3.0
highcharts-angular: 4.0.0
highcharts/map-collection: 2.0.0
angular version: 18.2.1

#### Affected browser(s)
Google Chrome

**Comment by oysteinmoseng** [comment]
Hello @pdaryamane,

The checker you are using is falsely reporting issues here. The data labels, such as "China" highlighted, have a white background and a black fill - however due to the way this is achieved in SVG, the accessibility checker is not picking this up correctly.

Also the `<desc>` element is not visible, and as such its contrast does not matter.

We recommend using a visual checking tool for checking contrast in SVGs, for example [CCA from TPGi](https://www.tpgi.com/color-contrast-checker/).

In general we try to ensure the default theme meets contrast standards, but please note that all colors can be changed through configuration depending on your particular use case. See [Design and Style](https://www.highcharts.com/docs/chart-design-and-style/design-and-style) as well as [Colors](https://www.highcharts.com/docs/chart-design-and-style/colors) in our documentation for more details.

Closing this for now, but feel free to comment to reopen if you have further details that require action from our side.

**Comment by pdaryamane** [comment]
Hi @oysteinmoseng : Thank you for the reply. I will check and get back if this needs to be reopened
